src/System.Private.Interpreter/src/System.Private.Interpreter.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyName>System.Private.Interpreter</AssemblyName>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
Could you make this conditioned on `Condition="'$(IsProjectNLibrary)' != 'true'"`, same as the other CSPROJ files? We need this to keep things buildable on the closed-source side.